kalite/distributed/management/commands/kaserve.py

```diff
@@ -53,7 +53,7 @@ class Command(BaseCommand):
             '--pidfile',
             action='store',
             dest='pidfile',
-            default=os.path.join(settings.PROJECT_PATH, "runcherrypyserver.pid"),
+            default=os.path.join(settings.USER_DATA_ROOT, "runcherrypyserver.pid"),
```

Comment:
Is the windows installer using the .pid file directly from a hard-linked path? @ruimalheiro ? But I would hope it wouldn't use it as `kalite start/stop/status` should deal with things...